### DISCUSSION TITLE
ALPS_paper

### DISCUSSION THREAD

**Discussion by MSAfify** [danielver02/ALPS#166] [Q&A] [answered]
Dear Prof. Daniel,
I prepared the following two files to generate Fig. 1a in the ALPS paper.  However, the output results differ from Figure 1a, for example, the output second root is just two points, and there is a difference between the scale of real and imaginary frequencies in Fig. 1a and the output results. Could you please inform me which variables should be set to generate the same figure 1a?
https://drive.google.com/file/d/1NnNwzWKMeFHWV_g14Ro2-Olmc-u2ECXu/view?usp=sharing
https://drive.google.com/file/d/1hPd8Gd2gjb52kBoTW4Ta6_WRCwdlDZmQ/view?usp=sharing

Thanks for your kind help!
Mahmoud

**Comment by kgklein** [comment, marked as answer]
Mahmoud-
Two items to note in your input file.
You are instructing ALPS to use the same guesses for the complex frequencies for both the Alfven and fast mode solution (your values for the namelists guess_1 and guess_2). I would try
&guess_1 !Alfven Solution guess
g_om=1.0d-02   !real frequency
g_gam=-2.d-07 !imaginary frequency
/
&guess_2 !Fast solution guess
g_om=2.0d-02   !real frequency
g_gam=-5.d-04 !imaginary frequency
/
As better initial guesses. Secondly, you may need to increase the variables ns or nres to ensure the solutions remain distinct as you scan through kparallel, as these two solutions can become quite close to one another.
Hope this helps, and let me know if you have other questions.
Kris

**Reply by MSAfify** [reply]
Hi Prof Kris,
Thanks so much!
Mahmoud
